ceph_deploy/util/templates.py

```diff
@@ -1,7 +1,6 @@
 
 
-ceph_repo = """
-[ceph]
```

Comment:
The suse/zypper one had already been changed, and I looked at it and said 'why is that one funny?'.  Then I ran the code and it was a 'duh' moment.  :)